src/cmd/manage_nsfs.js

```diff
@@ -366,6 +366,14 @@ async function fetch_account_data(argv, from_file) {
         const fs_context = native_fs_utils.get_process_fs_context();
         const raw_data = (await nb_native().fs.readFile(fs_context, from_file)).data;
         data = JSON.parse(raw_data.toString());
+        if (action === ACTIONS.ADD) {
+            if (data.creation_date) {
```

Comment:
hmmm I don't think we should ignore the user's creation date, what is your motivation here?

Reply:
The user doesn't define the `creation_date`, the manage nsfs CLI sets it (same as we don't have the flag to set it).
For example, now he can add that the account was created in the year 1800. I don't want to let him do it.

Reply:
hmm I was thinking about upgrade scenarios but I guess it's better to have the same behavior.
can you please add tests and maybe add to the documentation a json imput example of the from file case?

Reply:
Yes, I would plan to add.

A similar question about `allow_bucket_creation` - do we want to prevent from passing it in the file?

- If yes, then the JSON example will be changed (simple solution)
- If not
  - then we need to add a flag for this option (set and unset it) 
  - and remove the option to add a bucket when it's false.
  - change the required flag for account add (doesn't need to pass 'new_buckets_path' because this account doesn't create one)
  -etc... (too many changes for this PR scope, will need to open an issue).

Reply:
No, the user can decide whether the account can create buckets or not, why the JSON should be changed?

In a short glance on the NSFS account and bucket schemas there are similar cases to creation_date issue:
1. on NSFS bucket schema - 
    a. should_create_undelying_storage, this should on account add always be false, if the user provided true we should fail.
    b. _id - self generating
    c. bucket owner and owner_account should map to the same account ( this is an issue not only on --from_file flag) 
2. on nsfs account schema - 
    a. _id - self generating

Maybe there are more things like that, please check that as well

Reply:
If we prevent him from this then he cannot add this option (and the JSON would not contain the property `allow_bucket_creation`).
Currently, the manage nsfs CLI doesn't support adding this option and neither enforce it (we need to remove the option to add a bucket when it's false). I would open an issue.

Reply:
Currently, allow_bucket_creation option does not exist because its value is inferred based on the new_buckets_path value, we should have the same behavior on --from_file flow, but I don't see an issue with allowing it.
We can consider adding allow_bucket_creation explicitly to account add/update command.

We do not need to enforce it on the CLI, allow_bucket_creation is an option of a specific account while there is no specific account that runs the CLI commands.

Reply:
- Util we would add the option adding allow_bucket_creation explicitly to an account add/update command, the `from-file` should be aligned with it.
- By enforcing I meant that an account that has `allow_bucket_creation` `false` (account was created without `new_buckets_path`) should not be able to create a bucket (currently it can).

Reply:
1. I think that instead of removing an ability that we want in our code from the `from-file` flow and later bring it back, we can just add allow_bucket_creation flag to the arguments flow and use it or infer it on our own if it was not specified.

2. Creating a bucket is blocked if the account is not allowed to create a bucket, see here - https://github.com/noobaa/noobaa-core/commit/e995346640dc5e8026bf1932427677d63921b92c#diff-da078ac8593ed97975c9a5ed039a123ac81a0a1082f5645da1f62d74f9d83580L285
```
bucketspace_fs.create_bucket - 
if (!sdk.requesting_account.allow_bucket_creation) {
      throw new RpcError('UNAUTHORIZED', 'Not allowed to create new buckets')
}
```

Reply:
1. I think it would create bugs. I prefer a known gap, that an issue.
2. I agree with you and I understand it. I think you didn't understand me... so I would ask - Where is the equivalent part in `manage_nsfs` to bucket add command? (I think you will not find this in the code because it is a bug that I'm trying to point to...).

Reply:
regarding point 2 - got you now, so just add to verify_bucket_owner() another check that the owner is allowed to create buckets.

Reply:
Regarding point 1 - please see issue #7786.
Regarding point 2 - Please see PR #7785.

Reply:
@shirady @romayalon I think this will create some confusion [7786](https://github.com/noobaa/noobaa-core/issues/7786), What will happen when user provide `allow_bucket_creation` true without the `new-bucket-path`?